src/store/bundlr/reducer.ts

```diff
@@ -2,19 +2,14 @@ import { WebBundlr } from '@bundlr-network/client';
 import { Actions } from '..';
 import { ActionType } from './actions';
 
-export interface BundlrPendingBalance {
-  txId: string | null;
-  balanceBeforeFund: string;
-}
-
 export interface BundlrState {
   balance: string;
   bundlr: WebBundlr | null;
   isConnected: boolean;
   isFunding: boolean;
   isUploading: boolean;
   txId: string | null;
-  pendingBalance: BundlrPendingBalance;
+  balanceOffset: number;
```

Comment:
I'm not sure what format these Bundlr balances are in (I assume ETH?), but unless it's intentional, `balance` is a `string` and `balanceOffset` is a `number`.  It might be worth considering updating both types to `BigNumber`s, so that calculation and comparisons are done on a finer level.

Reply:
You're right, it would be better to use `BigNumber`. Thanks for catching that.